content/docs/04-clusters/01-public-cloud/03-azure/1-gateways.md

```diff
@@ -57,6 +57,12 @@ Log in to the Azure portal and create a Virtual Network taking care of the follo
     * Azure Kubernetes Cluster
     * Azure Bastion Host (Jump Box Virtual Machine) to connect to the Azure target Kubernetes cluster securely.
 
+<InfoBox>
+
+  **Note**: A bastion hosted is only required if accessing a Kubernetes cluster that is located inside a private Azure Virtual Network (VNet) that only exposes private endpoints. If you have direct network access to the PCG cluster then you do not require a bastion host. Alternatively, you can also deploy a bastion host and remove later if no longer required. 
+
+</InfoBox>
+
```

Comment:
<InfoBox>

 **Note**: A bastion host is only required while accessing a Kubernetes cluster inside a private Azure Virtual Network (VNet) exposed to private endpoints. You do not require a bastion host if you have direct network access to the PCG cluster. Alternatively, you can deploy a bastion host and remove it later if no longer required. 

</InfoBox>